src/icons/folder-icon/FolderIcon.tsx

```diff
@@ -1,30 +1,37 @@
 import * as React from 'react';
+import { FolderExternal, FolderPersonal, FolderShared } from '@box/blueprint-web-assets/icons/Content';
+import { useIntl } from 'react-intl';
 
-import IconFolderCollab from '../../icon/content/FolderShared32';
-import IconFolderExternal from '../../icon/content/FolderExternal32';
-import IconFolderPersonal from '../../icon/content/FolderPersonal32';
+import messages from '../../elements/common/messages';
 
-interface FolderIconProps {
+export interface FolderIconProps {
     /** Dimension of the icon */
     dimension?: number;
     /** If true displays collaborated folder icon */
     isCollab?: boolean;
     /** If true displays externally collaborated folder icon */
     isExternal?: boolean;
-    /** A text-only string describing the icon if it's not purely decorative for accessibility */
-    title?: string | React.ReactElement<string>;
```

Comment:
this is the same as for FileIcon. Admin Console uses `<FolderIcon />` directly and uses this prop